scripts/scil_visualize_bingham_fit.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Visualize 2-dimensional Bingham volume slice loaded from disk.
+
+Given an image of Bingham coefficients, this script displays a slice in a
+given orientation.
+"""
+
+import argparse
+
+import nibabel as nib
+import numpy as np
+
+from dipy.data import get_sphere
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.viz.scene_utils import (create_bingham_slicer,
+                                    create_scene, render_scene)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    # Positional arguments
+    p.add_argument('in_bingham', help='Input SH image file.')
+
+    # Window configuration options
+    p.add_argument('--slice_index', type=int,
+                   help='Index of the slice to visualize along a given axis. '
+                        'Defaults to middle of volume.')
+
+    p.add_argument('--win_dims', nargs=2, metavar=('WIDTH', 'HEIGHT'),
+                   default=(768, 768), type=int,
+                   help='The dimensions for the vtk window. [%(default)s]')
+
+    p.add_argument('--interactor', default='trackball',
+                   choices={'image', 'trackball'},
+                   help='Specify interactor mode for vtk window. '
+                        '[%(default)s]')
+
+    p.add_argument('--axis_name', default='axial', type=str,
+                   choices={'axial', 'coronal', 'sagittal'},
+                   help='Name of the axis to visualize. [%(default)s]')
+
+    p.add_argument('--silent', action='store_true',
+                   help='Disable interactive visualization.')
+
+    p.add_argument('--output', help='Path to output file.')
+
+    add_overwrite_arg(p)
+
+    sphere_choices = {'symmetric362', 'symmetric642', 'symmetric724',
+                      'repulsion724', 'repulsion100', 'repulsion200'}
+    p.add_argument('--sphere', default='symmetric362', choices=sphere_choices,
+                   help='Name of the sphere used to reconstruct SF. '
+                        '[%(default)s]')
```

Comment:
Is it possible for the default glyphs to look like a "multi-bingham distribution"? i.e. a color per bingham? As I saw before? 
![glyphs-bingham](https://user-images.githubusercontent.com/2170887/126626116-253f3ba2-83aa-4fdb-9cd6-dbdc134ee3fe.png)

Reply:
I added a `--color_per_lobe` option in d989322, which gives an output like this.
![image](https://user-images.githubusercontent.com/41654474/127340026-bc9c32b5-dfe3-45ba-8c90-ece2575ee513.png)